modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsAdd.es.js

```diff
@@ -206,6 +209,14 @@ function ResultRankingsAdd({cancelURL, fetchSitesURL, formName}) {
 								<div className="autofit-col-expand">
 									<div className="list-group-text text-dark">
 										{SCOPE_INFO[item].label}
+
+										{SCOPE_INFO[item].showBetaBadge && (
+											<ClayBadge
+												displayType="info"
+												label="beta"
+												translucent
+											/>
```

Comment:
Can you try using the BetaBadge component from `frontend-js-components-web`?

```
import {BetaBadge} from 'frontend-js-components-web';
```

Also 2 things about the styling:
1. Can you vertically center the beta badge and give it maybe 4px or 8px padding between the text and badge? It looks really misaligned in the screenshot.
![Screenshot 2023-12-21 at 11 28 35 AM](https://github.com/liferay-search/liferay-portal/assets/4496722/0c38e709-ffbb-4742-8178-5a87600e80e6)

2. Did the styling for the beta badge change? From the original beta badge docs the text is all caps. https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2312110481/Identifying+BETA+Features+in+the+product
![Screenshot 2023-12-21 at 11 47 45 AM](https://github.com/liferay-search/liferay-portal/assets/4496722/b2afe4d5-d466-4fd1-90f3-e52eeca9878c)

Reply:
@thektan Sure! I didn't know there was a Beta Badge already, I imported it but the badge is a bit bigger than the ClayBadge, also it doesn't look like it takes in additional styling? I was wondering how to fix the alignment that way.

<img width="1005" alt="Screenshot 2023-12-21 at 12 57 57 PM" src="https://github.com/liferay-search/liferay-portal/assets/14063502/b12916fb-245d-4ca2-ae11-d4e72e658575">

For the beta case, I'm actually not sure what is the standard, I did a search and it looked like several JS components around portal that kept the label "beta" lower case. It looks like the frontend BetaBadge is just capitalized, too 🤔 ... should we change to all caps?

Reply:
You can center doing something like (but make it scoped to the ClayDropDown.Menu)

```css
.list-group-text {
  align-items: center;
  display: flex;
}
```

In that case, we can continue to use `ClayBadge` instead, but let's uppercase using `className="text-uppercase"` too.

Reply:
Added the updates! Lmk if that looks ok 👍  
<img width="970" alt="Screenshot 2023-12-21 at 3 34 13 PM" src="https://github.com/liferay-search/liferay-portal/assets/14063502/08a10c43-67b0-4d2c-b17f-baf9a403ac96">

Reply:
Yup much better! Thanks! 💯